tests/auth_tests/test_forms.py

```diff
@@ -221,11 +224,11 @@ def test_duplicate_normalized_unicode(self):
     )
     def test_validates_password(self):
         data = {
-            "username": "testclient",
-            "password1": "testclient",
-            "password2": "testclient",
+            "username": "toastclient",
+            "password1": "toastclient",
+            "password2": "toastclient",
```

Comment:
This change is a noop for `BaseUserCreationForm` (which does not check for username uniqueness), but is needed for `UserCreationForm` which defines `clean_username` and ensures that the username is unique (and `setUpTestData` already creates a user with username `testclient`).

Reply:
This looks like a typo, can this be `"newclient"` or `"otherclient"`?

Reply:
It was not a typo, I was thinking of "make toast", but happy to change it! 🙂

Reply:
I think my personal preference is `"testuser"`